README.md

```diff
@@ -161,33 +161,44 @@ Fields to return:
 - `available_inventory` (not currently checked-out to a customer)
   - This will be the same as `inventory` unless you've completed the optional endpoints.
 
+#### `POST /movies`
+Create a new movie in the video store inventory.
+
+Request body:
+
+| Field         | Datatype            | Description
+|---------------|---------------------|------------
+| `title` | string             | Title of the movie
+| `overview` | string | Descriptive summary of the movie
+| `release_date` | string `YYYY-MM-DD` | Date the movie was released
+| `inventory` | integer | Quantity available in the video store
+
 ### Optional Rentals
 
 Wave 2 focused on **reading** data from the API.  In these endpoints you will interact with the API changing models through POST requests.  
```

Comment:
This is no longer accurate since we now have an endpoint for creating a new movie. We should probably rephrase like: "Wave 2 focused on working with customers and movies. With these endpoints you can extend the functionality of your API to allow managing the rental process."